models/prices/prices_tokens.sql

```diff
@@ -1031,10 +1031,90 @@ VALUES
     ("usdt-tether", "gnosis", "USDT", "0x4ecaba5870353805a9f068101a40e0f32ed605c6", 6),
     ("wbtc-wrapped-bitcoin", "gnosis", "WBTC", "0x8e5bbbb09ed1ebde8674cda39a0c169401db4252", 8),
 
+    ("eth-ethereum", "optimism", "ETH", "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee", 18),
+    ("eth-ethereum", "optimism","ETH", "0xdeaddeaddeaddeaddeaddeaddeaddeaddead0000",18),
     ("dai-dai", "optimism", "DAI", "0xda10009cbd5d07dd0cecc66161fc93d7c9000da1", 18),
     ("usdc-usd-coin", "optimism", "USDC", "0x7f5c764cbc14f9669b88837ca1490cca17c31607", 6),
     ("usdt-tether", "optimism", "USDT", "0x94b008aa00579c1307b0ef2c499ad98a8ce58e58", 6),
     ("wbtc-wrapped-bitcoin", "optimism", "WBTC", "0x68f180fcce6836688e9084f035309e29bf0a2095", 8),
+    ("weth-weth","optimism","WETH","0x4200000000000000000000000000000000000006",18),
+    ("hav-havven","optimism","SNX","0x8700daec35af8ff88c16bdf0418774cb3d7599b4",18),
+    ("link-chainlink","optimism","LINK","0x350a791bfc2c21f9ed5d10980dad2e2638ffa7f6",18),
+    ("rgt-rari-governance-token","optimism","RGT","0xb548f63d4405466b36c0c0ac3318a22fdcec711a",18),
+    ("mkr-maker","optimism","MKR","0xab7badef82e9fe11f6f33f87bc9bc2aa27f2fcb5",18),
+    ("uni-uniswap","optimism","UNI","0x6fd9d7ad17242c41f7131d257212c54a0e816691",18),
+    ("rai-rai-reflex-index","optimism","RAI","0x7fb688ccf682d58f86d7e38e03f9d22e7705448b",18),
+    ("susd-susd","optimism","sUSD","0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9",18),
+    ("seth-seth","optimism","sETH","0xe405de8f52ba7559f9df3c368500b6e6ae6cee49",18),
+    -- ("NULL","optimism","sLINK","0xc5db22719a06418028a40a9b5e9a7c02959d0d08",18),
+    -- ("NULL","optimism","sBTC","0x298b9b95708152ff6968aafd889c6586e9169f1d",18),
+    ("reth-rocket-pool-eth","optimism","rETH","0x9bcef72be871e61ed4fbbc7630889bee758eb81d",18),
```

Comment:
remove or comment out tokens with inactive price feeds on coinpaprika

Reply:
on it